students/toddschultz/session02/print_grid.py

```diff
@@ -0,0 +1,74 @@
+
+# PART ONE
+verticle = "+ - + - +"
+horizontal = "|   |   |"
+
+print (verticle + "\n" + horizontal + "\n" + verticle + "\n" + horizontal + "\n" + verticle)
+
+
+
+# PART TWO
+#The assignment shows 4 on each side for 8, and 7 on each side for 15.
+#therefore, if it's an odd number, I remove one.
+def print_grid(n):
+	if n % 2 == 0:
```

Comment:
make sure your editor is set to 4 spaces (not tabs) for indentation. this is currently over-indented.

something like pycharm can do this for you automatically.